framework/src/postprocessors/FunctionValuePostprocessor.C

```diff
@@ -18,14 +18,22 @@ FunctionValuePostprocessor::validParams()
   InputParameters params = GeneralPostprocessor::validParams();
   params.addRequiredParam<FunctionName>("function",
                                         "The function which supplies the postprocessor value.");
+
   params.addParam<Point>(
       "point", Point(), "A point in space to be given to the function Default: (0, 0, 0)");
+  params.addParam<std::vector<PostprocessorName>>(
+      "point_postprocessors",
+      "A tuple of Postprocessors that are passed to the function as the space "
+      "argument. Cannot be specified if point argument is specified.");
   params.addParam<Real>("scale_factor", 1, "A scale factor to be applied to the function");
   params.addParam<std::vector<PostprocessorName>>(
       "indirect_dependencies",
       "If the evaluated function depends on other postprocessors they must be listed here to "
       "ensure proper dependency resolution");
 
+  params.addParam<PostprocessorName>(
```

Comment:
If you can fold bot options into a `point` parameter this one should just be called `time`.